os/mm/mm_heap/mm_heapinfo_parse_heap.c

```diff
@@ -64,11 +64,109 @@
 #define HEAPINFO_INT INT16_MAX
 #define HEAPINFO_NONSCHED (INT16_MAX - 1)
 
+/****************************************************************************
+ * Public data
+ ****************************************************************************/
+
+typedef enum
+{
+	TYPE_CORRUPTED,
+	TYPE_OVERFLOWED,
+} node_type_t;
```

Comment:
In TizenRT, we declare the enum type with `xxx_e` at the end.
So, please change the type name to `node_type_e` instead of `node_type_t`.

Reply:
_t is correct for typedef. But our coding rule recommends splitting the enum definition and type definition.

Reply:
Fixed.